### PR TITLE
Bump android dropin and fix 3Ds (#29)

### PR DESCRIPTION
**Pull request by luke-hawk**
Ok, so first, sorry for not having checked android properly in my last PR and hence issue #29. This should fix it now. I've bumped the dropin version and fixed and issue related to `collectDeviceData` (see comment). Tested both in the demo project and everything works well: We get a redirect to 3Ds and device data returned with the nonce. 

Would be great if you could give it a quick check and then publish a new version :-)

Cheers!

### CODE REVIEW

File: src/braintree.android.ts [hunk 1]
```diff
@@ -37,16 +37,12 @@ export class Braintree extends Observable {
 
         let clientTokenMethod = dropInRequest.getClass().getMethod("clientToken", [java.lang.String.class]);
 
-        let isAndroidPayEnabled = dropInRequest.getClass().getMethod("isAndroidPayEnabled", []);
-        let booleanType = isAndroidPayEnabled.getReturnType();
-
         if (options.collectDeviceData) {
-            let collectDeviceDataMethod = dropInRequest.getClass().getMethod("collectDeviceData", [booleanType]);
-            collectDeviceDataMethod.invoke(dropInRequest, [true]);
```
Comment (jibon57) [inline src/braintree.android.ts:45]: Before NativeScript Android runtime had bug with metadata generation that's why I have used reflection. You can get more details from here: https://github.com/NativeScript/android-runtime/issues/831#issuecomment-325651230
So, now it's working without reflection?

Comment (luke-hawk) [reply src/braintree.android.ts:45]: Yeah, I found the issue with the runtime bug. It seems to work without the reflection now, yes! I've tested everything, but would appreciate if you'd double check ;)

**Comment by luke-hawk** [comment]
Ok, so apparently there still seems to be a problem with iOS:

```
[com.apple.Registration:ServiceAvailability] <FTCServiceAvailabilityCenter: 0x6000011b8080> adding listener `MFMessageComposeViewController:org.nativescript.demo#17B13166-FA14-41F7-858F-AAD4DFA17DB9` for service: iMessage
```

I'll look into this later today and will add another commit to fix it, so please not merge the PR yet!

**Comment by luke-hawk** [comment]
Ok, so now finally everything should work as expected. Sorry for the troubles!

**Comment by luke-hawk** [comment]
@jibon57 Have you had a chance to look at it yet?

**Comment by jibon57** [comment]
Sorry for long delay.

**Pull request merged**